src/battle_script_commands.c

```diff
@@ -6200,6 +6200,34 @@ static void Cmd_moveend(void)
                 gBattleStruct->sameMoveTurns[gBattlerAttacker]++;
             gBattleScripting.moveendState++;
             break;
+        case MOVEEND_SET_EVOLUTION_TRACKER:
+            // If the Pokémon needs to keep track of move usage for its evolutions, do it
+            if (originallyUsedMove != MOVE_NONE
+                && GetBattlerSide(gBattlerAttacker) == B_SIDE_PLAYER
+                && !(gBattleTypeFlags & (BATTLE_TYPE_LINK
```

Comment:
```suggestion
+            if (originallyUsedMove != MOVE_NONE
+             && GetBattlerSide(gBattlerAttacker) == B_SIDE_PLAYER
+             && !(gBattleTypeFlags & (BATTLE_TYPE_LINK
```
Could we agree on this formatting? I know there are a lot of inconsistencies but for new stuff we might as well try to keep it consistent.

Reply:
I absolutely and positively agree. *Yes, I'm biased, how could you tell?*